### PR TITLE
Update profileCard.tsx

### PR DESCRIPTION
**Pull request by GradleD**
Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix : Updated the Specific Profile Card with skeleton Loading component

## Summary by CodeRabbit

- **New Features**
	- Introduced a loading skeleton UI in the ProfileCard component to enhance user experience during data fetching.
	- Added a new prop, `isLoading`, to indicate the loading state of the ProfileCard.
	- Streamlined loading logic for portfolio summaries, displaying a skeleton UI when data is being fetched.

These changes aim to provide users with a smoother experience while waiting for data to load.

### CODE REVIEW

File: components/UI/profileCard/profileCard.tsx [hunk 1]
```diff
@@ -120,38 +123,55 @@ const ProfileCard: FunctionComponent<ProfileCardProps> = ({
...
+      {/* Updated the Specific Profile Card Component with skeleton Loading component */}
+      <Skeleton variant="text" width="80%" height={20} />
+      <Skeleton variant="text" width="60%" height={30} className="mt-2" />
+      <div className={styles.address_div}>
+        <div className="flex items-center gap-2">
+          <Skeleton variant="rectangular" width={100} height={30} />
+          <EyeIcon />
+        </div>
+      </div>
+    </>
```
Comment (fricoben) [inline components/UI/profileCard/profileCard.tsx]: Create a specific component skeleton in the `/skeleton` folder

File: components/UI/profileCard/profileCard.tsx [hunk 2]
```diff
@@ -36,6 +37,7 @@ type ProfileCardProps = {
   identity: Identity;
   leaderboardData: LeaderboardToppersData;
   isOwner: boolean;
+  isLoading:boolean;
```
[the remark this answers is not in the document]
Comment (fricoben) [reply components/UI/profileCard/profileCard.tsx:40]: You add is Loading to the component props but you don't add it where the component is used + you unforce it. 

Please test the stuff you send before a PR

**Review by fricoben — CHANGES_REQUESTED** [review]
1. It doesn't build because you didn't use the props correctly (this shows you didn't test what you're sending)
2. Please create a specific component for the skeleton

File: app/[addressOrDomain]/page.tsx [hunk 3]
```diff
@@ -289,6 +294,10 @@ export default function Page({ params }: AddressOrDomainProps) {
         assetValues[symbol] = (assetValues[symbol] || 0) + value;
       }
     });
+
+
+
+
```
Comment (fricoben) [inline app/[addressOrDomain]/page.tsx:300]: delete useless space

File: app/[addressOrDomain]/page.tsx [hunk 4]
```diff
@@ -103,6 +105,8 @@ export default function Page({ params }: AddressOrDomainProps) {
     []
   );
 
+
+
```
Comment (fricoben) [inline app/[addressOrDomain]/page.tsx:109]: useless space again please use prettier vs code extension to format everything

File: app/[addressOrDomain]/page.tsx [hunk 5]
```diff
@@ -83,6 +83,8 @@ export default function Page({ params }: AddressOrDomainProps) {
   const [identity, setIdentity] = useState<Identity>();
   const [notFound, setNotFound] = useState(false);
   const [isOwner, setIsOwner] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
```
Comment (fricoben) [inline app/[addressOrDomain]/page.tsx:87]: useless space + unprecise naming, what is loading here ? I would go for `isProfileCardLoading`

File: app/[addressOrDomain]/page.tsx [hunk 6]
```diff
@@ -585,6 +602,10 @@ export default function Page({ params }: AddressOrDomainProps) {
     setInitProfile(false);
   }, [address, addressOrDomain]);
 
+
+
+
+
```
Comment (fricoben) [inline app/[addressOrDomain]/page.tsx:608]: useleszs space again use prettier !

**Review by fricoben — CHANGES_REQUESTED** [review]
1. Lots of code improvements 
2. It's still not working, I don't see any skeleton on my profile card it has exactly the same problem than before the issue was created. 

Capture d’écran 2024-11-12 à 13 25 59

Test before sending a commit on github cause we're both loosing time here, I'll review one more time. If it's not working on my next review I'll need to give this issue to someone else :/

**Comment by wheval** [comment]
@fricoben if he can't handle this, i would love to help fix this. Thanks.

**Comment by martinvibes** [comment]
Hey @fricoben I would love and help to work on this if he is not able to finish the task 
Thank ya

**Comment by GradleD** [comment]
@fricoben  Kindly assign someone else

**Comment by fricoben** [comment]
Will assign to someone else

**Pull request closed without merging**